File: tests/fixtures/bad.go
// Deliberate-bad sample file for cyscan integration tests.
package main

import (
	"crypto/md5"
	"crypto/tls"
	"fmt"
)

func run() {
	cfg := &tls.Config{InsecureSkipVerify: true} // CBR-GO-TLS-INSECURE-SKIP-VERIFY
	fmt.Println(cfg, md5.New())                   // CBR-GO-WEAK-HASH
}

File: tests/fixtures/bad.py
# Deliberate-bad sample file for cyscan integration tests.
# Every block below should raise at least one finding.

import sqlite3

AWS_ACCESS_KEY = "AKIA1234567890ABCDEF"  # CBR-SECRETS-AWS-KEY

def lookup(conn, email):
    cursor = conn.cursor()
    # CBR-PY-SQLI-STRING-CONCAT
    cursor.execute("SELECT * FROM users WHERE email = '" + email + "'")
    return cursor.fetchall()
